### ISSUE TITLE
Update the display in the animation frame

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/papyrus#23]
It currently updates at a slow rate, would be nice to update the display in the animation frame.

**Comment by jbphet** [comment]
@jessegreenberg - I think you've implemented this, correct?

**Comment by jessegreenberg** [comment]
Right! Sorry for the duplication, this was done in https://github.com/phetsims/papyrus/issues/32. Closing.
